src/datachain/lib/dc.py

```diff
@@ -577,6 +572,43 @@ def datasets(
             **{object_name: datasets},  # type: ignore[arg-type]
         )
 
+    @classmethod
+    def listings(
+        cls,
+        session: Optional[Session] = None,
+        in_memory: bool = False,
+        object_name: str = "listing",
+        **kwargs,
+    ) -> "DataChain":
+        """Generate chain with list of cached listing.
+
+        Example:
+            ```py
+            from datachain import DataChain
+
+            chain = DataChain.listings()
```

Comment:
can we use `show()` instead? makes it simpler

Reply:
Yes, that's good idea